docs/integrations/sources/jenkins.md

```diff
@@ -0,0 +1,58 @@
+# Jenkins
+
+## Overview
+
+The Jenkins source is maintained by [Faros
+AI](https://github.com/faros-ai/airbyte-connectors/tree/main/sources/jenkins-source)
+and supports both Full Refresh and Incremental syncs. You can choose if this
+connector will copy only the new or updated data, or all rows in the tables and
+columns you set up for replication, every time a sync is run.
+
+### Output schema
+
+Several output streams are available from this source:
+
+* [Builds](TODO) \(Incremental\)
```

Comment:
@Roma-Kyrnis Can you provide links to Jenkins API docs here?

Reply:
Ok @cjwooo I'll add

Reply:
Can you update to?
Set `JOB_NAME` and `BUILD_NUMBER` to accept Jobs and Builds API schema
* [Builds]($JENKINS_SERVER_URL/job/$JOB_NAME/$BUILD_NUMBER/api/json?pretty=true) \(Incremental\)
* [Jobs]($JENKINS_SERVER_URL/job/$JOB_NAME/api/json?pretty=true)